logger_config.py

```diff
@@ -1,17 +1,17 @@
+from functools import wraps
 import logging
 FORMAT = '%(asctime)s - %(name)-s - %(levelname)-s - %(message)s'
 LEVEL = logging.DEBUG
 logging.basicConfig(format=FORMAT, level=LEVEL)
 log = logging.getLogger(__name__)
 
+
 def logger(fn):
-    from functools import wraps
-    import inspect
     @wraps(fn)
     def wrapper(*args, **kwargs):
         log = logging.getLogger(fn.__name__)
         log.info('About to run %s' % fn.__name__)
-        out = apply(fn, args, kwargs)
+        out = fn(*args, **kwargs)
```

Comment:
apply() call has been changed to fn() - is this intended?

Reply:
This has been modified in the latest PR. Please pull the latest code, do not make this change @Mansi145

Reply:
Yes, it was intended but I'll revert the changes.

Reply:
The apply() function does not work in Python 3. It has been replaced by fn(). You can see the same changes in your recent merge.